testsuite/ui/browser.py

```diff
@@ -104,3 +107,24 @@ def new_tab(self, trigger, keep_tab=False):
         else:
             self.browser.close_window(new_handle)
             self.browser.switch_to_window(current_handle)
+
+    @backoff.on_exception(backoff.fibo, NoSuchElementException, max_tries=13, jitter=None)
```

Comment:
I think max_tries=13 is too much. Use e.g. 8 as in rest of testsuite.

Reply:
8 was sometimes not enough for the OTP form to load.